cpg_workflows/workflow.py

```diff
@@ -1317,27 +1253,6 @@ def queue_jobs(self, dataset: Dataset, inputs: StageInput) -> StageOutput | None
         """
         pass
 
-    def deprecated_queue_for_cohort(self, cohort: Cohort) -> dict[str, StageOutput | None]:
-        """
-        Plug the stage into the workflow.
-        unused, ready for deletion
-        """
-        output_by_target: dict[str, StageOutput | None] = dict()
-        if not (datasets := cohort.get_datasets()):
-            logging.warning(
-                f'{len(cohort.get_datasets())}/'
-                f'{len(cohort.get_datasets(only_active=False))} '
-                f'usable (active=True) datasets found in the cohort. Check that '
-                f'`workflow.input_datasets` or `workflow.input_cohorts` is provided, and not all datasets are skipped '
-                f'via workflow.skip_datasets`',
-            )
-            return output_by_target
-        for dataset_i, dataset in enumerate(datasets):
-            action = self._get_action(dataset)
-            logging.info(f'{self.name}: #{dataset_i + 1}/{dataset} [{action.name}]')
-            output_by_target[dataset.target_id] = self._queue_jobs_with_checks(dataset, action)
-        return output_by_target
```

Comment:
As discussed with @vivbak - this method (and all other `deprecated_queue_for_cohort` instances) are unused in `main` and scheduled for removal already. 

Their removal here is not just housekeeping - they would have to be re-written in light of the other other changes in this PR (Cohorts don't have Datasets), so we'd need to renovate these unused methods just to prevent the code from causing failures during workflow setup. On that basis they're being bundled into the broader change.